pkg/coordinator/handlers.go

```diff
@@ -217,7 +217,9 @@ func (o *Server) WS(w http.ResponseWriter, r *http.Request) {
 
 	// Assign available worker to browserClient
 	bc.WorkerID = wc.WorkerID
-	wc.IsAvailable = false
+
+	wc.ChangeUserQuantityBy(1)
```

Comment:
btw, you can try this from golang : D. It supports atomic computation. https://golang.org/pkg/sync/atomic/

Reply:
I don't like its syntax: readandgetputInt345. Also, CAS instructions can be quite tricky. Already used it before in the cloud-game code, doesn't matter performance-wise for this chunk of code.